docs/developers/helm-charts.md

```diff
@@ -0,0 +1,53 @@
+---
```

Comment:
when adding new pages, `mkdocs.yml` should be updated too.
https://celeborn.apache.org/community/contributor_guide/docs_and_website/

Reply:
@pan3793 Updated.